press/views/ping.py

```diff
@@ -3,19 +3,14 @@
 from pyramid.view import view_config
 
 
-@view_config(route_name='ping', http_cache=0)
-def ping(request):
-    """A ping and ack view for checking the service is up and running."""
-    return Response('pong')
-
-
+@view_config(renderer='text', route_name='ping', http_cache=0)
+@view_config(route_name='api-ping', http_cache=0)
 @view_config(route_name='auth-ping', http_cache=0, permission='view')
-def authedping(request):
-    """Authenticated ping for testing the auth mechanism."""
-    return Response('pong')
-
-
 @view_config(route_name='publish-ping', http_cache=0, permission='publish')
-def publish_ping(request):
-    """Authenticated ping for testing the publish endpoint"""
-    return Response('pong')
+def ping(request):
+    """A ping and ack view for checking the service is up and running.
+
+    There are multiple views to accommodate multiple paths
+    and different permissions"""
+    return Response('pong {path}'.format(path=request.path),
+                    content_type='text/plain')
```

Comment:
`@view_config(renderer='text', ...)` would mean you could just `return 'pong {path}'.format(path=request.path)`

It'll be fine, but why not use `renderer='text'` for all the view_config declarations rather than just one?

Reply:
Because when it errors it sends json. Wasn't sure if the content_type would be overridden properly.

Reply:
Hmm, perhaps it would 'just work' :-)

Reply:
And it does! checked and tested. (Seems we didn't have a `/ping` test, since there _is_ no `text` renderer, it's named `string`)